src/test/optimizer/strategy/in_expression_rewrite_rule_test.cpp

```diff
@@ -80,34 +80,40 @@ TEST_F(InExpressionRewriteRuleTest, DisjunctionStrategy) {
     const auto input_lqp = PredicateNode::make(five_element_in_expression, node);
```

Comment:
Not yours, but `five_element` misses an s.

Reply:
I wouldn't say so. It is a five-element InExpression. Shouldn't matter, anyway.